explainaboard/metrics/regression.py

```diff
@@ -0,0 +1,359 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import cast, Optional, Union
+
+import numpy as np
+from scipy.stats import pearsonr
+
+from explainaboard.metrics.metric import Metric, MetricConfig, MetricStats
+from explainaboard.metrics.registry import register_metric_config
+
+
+@dataclass
+@register_metric_config
+class SegKtauCorrConfig(MetricConfig):
```

Comment:
(1) Could `no_human` and `threshold` be documented?
(2) Maybe we should consider the default for `threshold` being `0`? `25` seems a bit arbitrary and opaque.
(3) It'd be nicer if we could avoid having `Seg` be part of the class name, and rather define `corr_level = 'seg'` or `corr_level = 'sys'` as a variable in the configuration. That would make it easier to calculate `Ktau`, `Pearson`, `Spearman` etc. with respect to either segment-level or system-level correlation.

Reply:
fixed: `Regarding (1) (2) will fix them once I get feedbacks from Yiwei`